shapely/predicates.py

```diff
@@ -596,9 +596,8 @@ def contains(a, b, **kwargs):
 
 @multithreading_enabled
 def contains_properly(a, b, **kwargs):
-    """Return True if geometry B is completely inside geometry A.
-
-    They should have no common boundary points.
+    """Return True if geometry B is completely inside geometry A, with no common
```

Comment:
@theroggy thanks for looking after the docstrings!

Have you ever seen an open source tool for checking the *prose* of a docstring? The actual writing style?

I've aspired to never begin with "Return ..." or "Returns ..." because returning (on success) is in the very nature of a Python method and we have a numpy "Returns" section specifically for that. I must say that I don't always live up to my aspirations and so a tool would be handy.

I also prefer a first line that is as terse as possible, as per https://numpydoc.readthedocs.io/en/latest/format.html#short-summary.

Reply:
> I also prefer a first line that is as terse as possible, as per https://numpydoc.readthedocs.io/en/latest/format.html#short-summary.

FWIW, the change was because my explicit ask to make it longer again ..

I fully agree that ideally the first line is as short as possible and to generally follow the rule of a single line summary. However, I also think that summary should still be as clear as possible, and so I commented on the previous PR that IMO in this case "practicality beats purity" and asked to expand that first sentence, to make it more explicit what the exact difference is of "contains properly" (i.e. it's the part of "no common boundary points" that differentiates it from `contains`)

Reply:
> Have you ever seen an open source tool for checking the _prose_ of a docstring? The actual writing style?

Ruff, via the [pydocstyle rules](https://docs.astral.sh/ruff/rules/#pydocstyle-d) does check the writing style recommendations that are in [PEP 257](https://peps.python.org/pep-0257/) e.g.:
- The first line/summary of the docustring should be one line: [blank-line-after-summary](https://docs.astral.sh/ruff/rules/blank-line-after-summary/)
- The summary should be in an "imperative mood". So e.g. "Returns..." is not allowed, it should be "Return...": [non-imperative-mood](https://docs.astral.sh/ruff/rules/non-imperative-mood/)
- The summary should not start with "This": [docstring-starts-with-this](https://docs.astral.sh/ruff/rules/docstring-starts-with-this/)
- ...
 
> I've aspired to never begin with "Return ..." or "Returns ..." because returning (on success) is in the very nature of a Python method and we have a numpy "Returns" section specifically for that. I must say that I don't always live up to my aspirations and so a tool would be handy.

I do understand your line of thinking, but """Return the pathname of the KOS root directory.""" is actually the example in PEP 257 for a [one line docstring](https://peps.python.org/pep-0257/#one-line-docstrings)... so at least the authors of PEP 257 were OK with that. Hence, I doubt that there will be linting tools that explicitly check for this out of the box...